packages/env-utils/src/envUtils.native.ts

```diff
@@ -73,6 +73,15 @@ const getOsNameWeb = () => '';
 
 const getOsFamily = (): 'Linux' => 'Linux';
 
+/*
+ * On the app launch and then about once a minute (MESSAGES_FETCH_INTERVAL), a new config tries to fetch.
+ * MESSAGES_CHECK_INTERVAL is interval for checking if MESSAGES_FETCH_INTERVAL already expired.
+ * in miliseconds
+ */
+
+const MESSAGES_FETCH_INTERVAL = 600_000;
```

Comment:
This doesn't belong to env-utils, this should be in message package. You can do `MESSAGES_CHECK_INTERVAL = isMobile() ? 100000 : 50000` etc

Reply:
3e7398770